Sources/MapboxCoreNavigation/NavigationSettings.swift

```diff
@@ -347,6 +348,18 @@ public class NavigationSettings {
 
     init() {
         setupFromDefaults()
+        SdkInfoRegistryFactory.getInstance().registerSdkInformation(forInfo: .init(
```

Comment:
Should Directions, Speech, etc also be updated to call `registerSdkInformation` on Common?